pkg/internal/client/client.go

```diff
@@ -49,7 +55,27 @@ func New() (*Client, error) {
 		return nil, err
 	}
 
-	return &Client{client}, nil
+	cniClient, err := netattdefclient.NewForConfig(config)
+	if err != nil {
+		return nil, err
+	}
+
+	return &Client{client, cniClient}, nil
+}
+
+func (c *Client) GetNetworkAttachmentDefinition(
+	ctx context.Context,
+	namespace, name string) (*netattdefv1.NetworkAttachmentDefinition, error) {
+	return c.K8sCniCncfIoV1Interface.NetworkAttachmentDefinitions(namespace).Get(ctx, name, k8smetav1.GetOptions{})
+}
+
+func (c *Client) CreatePod(ctx context.Context, namespace string, pod *k8scorev1.Pod) (*k8scorev1.Pod, error) {
+	log.Printf("Creating Pod %s/%s..", namespace, pod.Name)
+	return c.KubevirtClient.CoreV1().Pods(namespace).Create(ctx, pod, k8smetav1.CreateOptions{})
+}
+
+func (c *Client) GetPod(ctx context.Context, namespace, name string) (*k8scorev1.Pod, error) {
+	return c.KubevirtClient.CoreV1().Pods(namespace).Get(ctx, name, k8smetav1.GetOptions{})
 }
```

Comment:
Could you please add the pod functionality in a different PR?

Reply:
DONE